scripts/log_parser.py

```diff
@@ -288,6 +288,17 @@ def plot(self) -> None:
         plt.show()
 
 
+def extract_gc_objects_set(log_file_path):
```

Comment:
Note: for matplotlib to render the plots, at least in Linux I'm required to have pyqt5 installed from pypi. I have not added it to our pipfile as I felt it was a bit overkill for just using a simple parser from time to time.

Reply:
You might want to add a relevant comment in the script so its known to future users.